recipes-core/systemd/systemd-conf_%.bbappend

```diff
@@ -0,0 +1,4 @@
+do_install:prepend() {
+	# restore old match to match only on management ports
+	sed -i 's/Name=!veth\*/Name=en\* eth\*/' ${WORKDIR}/wired.network
+}
```

Comment:
The commit message says the changed behavior "is obviously a bad idea on switches". I don't disagree, but it would have been nice to sketch out the consequences.

Reply:
Do you want me to spell out that this starts a DHCP client on all ports, or what is your expectation?

Reply:
Clarified a bit.

Reply:
```
    +    This will start a DHCP client on all front panel ports, which is
    +    obviously a bad idea on switches, which should accept any configuration
    +    on those ports unless explicitly configured.
```

Did you mean to write "which should _not_ accept any configuration"!?

Reply:
Yes. Fixed.